src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -54,17 +59,17 @@ class PressableWithSecondaryInteraction extends Component {
         // On Web, Text does not support LongPress events thus manage inline mode with styling instead of using Text.
         return (
             <Pressable
-                style={this.props.inline && styles.dInline}
+                style={StyleUtils.combineStyles(this.props.inline ? styles.dInline : this.props.style)}
                 onPressIn={this.props.onPressIn}
-                onLongPress={(e) => {
+                onLongPress={this.props.onSecondaryInteraction ? (e) => {
```

Comment:
This can be more readable if we extract this into a function and have something like this.props.onSecondaryInteraction ? executeSecondaryInteraction : undefined